src/chrome/content/rules/ABC-Online.xml

```diff
@@ -3,48 +3,21 @@
 
 		- ABC-Music-Publishing.xml
 
-
-	CDN buckets:
-
-		- d1ros97qkrwjf5.cloudfront.net
-		- d3mfbaa198drag.cloudfront.net
-		- cp44823.edgefcs.net
-
-
-	Nonfunctional:
-
-		- about.abc.net.au ¹
-		- mpegmedia.abc.net.au ²
-		- origin.abc.net.au ³		(redirects to www via http)
-		- (www.)abc.net.au ²
-		- www2b.abc.net.au ³
-		- (www.)abccommercial.com.au ³
-		- (www.)abccontentsales.com.au ³
-		- (www.)abcdigmusic.net.au
-		- (www.)radioaustralianews.net.au ³
-		- (www.)triplejunearthed.com ³
-
-	¹ 503, akamai
-	² 504, akamai
-	³ Dropped
-
+	Non-functional hosts in *.abc.net.au
+		Timeout:
+		- abc.net.au
```

Comment:
This target works fine for me. Could you double check please?

Reply:
If travis is the only issue, I think we should whitelist this ruleset. The target looks fine on ssllabs: https://www.ssllabs.com/ssltest/analyze.html?d=abc.net.au